eva/catalog/catalog_manager.py

```diff
@@ -249,6 +249,10 @@ def drop_udf(self, udf_name: str) -> bool:
         """
         return self._udf_service.drop_udf_by_name(udf_name)
 
+    def rename_table(self, new_name: str, metadata_id: int):
```

Comment:
We have stopped working with ids and rather pass the catalog object

Reply:
This was left by the students who added support of rename. Please fix it to use the current approach.